proposal/success.Rmd

```diff
@@ -36,5 +36,5 @@ What sort of things could come up that can delay or break the project?
  - [ ] Costs
 
 -->
-This is a low risk project. There is already a base of material to build on, from workshops delivered by Hadley Wickham and course material from Jenny Bryan and Di Cook used in University classes. There is little risk that the new materials would not be developed. We are expecting for some young women to travel to workshops, and there are always risks associated with travel. It may be necessary for instructors and tutors to have police background checks in order to work with high school students, and this needs to be organised with the local committees.
+This is a low risk project. There is already a base of material to build on, from workshops delivered by Hadley Wickham and course material from Jenny Bryan and Di Cook used in University classes. There is little risk that the new materials would not be developed. We are expecting for some participants to travel to workshops, and there are always risks associated with travel. It may be necessary for instructors and tutors to have police background checks in order to work with high school students, and this needs to be organised with the local committees.
```

Comment:
I find the comment about travel being risky to be odd. I almost removed it but didn't. In any case, I don't think there is anything special about "young women" travelling, so I did rephrase that.